example/tests/integration/test_includes.py

```diff
@@ -20,6 +20,21 @@ def test_included_data_on_list(multiple_entries, client):
     assert comment_count == expected_comment_count, 'List comment count is incorrect'
 
 
+def test_included_data_on_list_with_one_to_one_relations(multiple_entries, client):
+    response = client.get(reverse("entry-list"),
+                          data={'include': 'authors.bio.metadata', 'page[size]': 5})
+    included = response.json().get('included')
+
+    assert len(response.json()['data']) == len(multiple_entries), (
+        'Incorrect entry count'
+    )
+    expected_include_types = ['authorBios', 'authorBios', 'authors', 'authors']
```

Comment:
when `authors.bio.metadata` is requested to be included shouldn't there be a authorsBioMetadata type be included as well?